store/todo/actions.js

```diff
@@ -0,0 +1,8 @@
+export default {
+    addTask({ state, commit }){
+      commit('setTask', val)
+    },
+    addFilteredList({state,commit}){
+      commit('setFilteredList',val)
+    }
+  }
```

Comment:
Please consider the pros and cons of using separate files for various actions, getters, state etc.

Reply:
Resolved